exec/RenderHtml.hs

```diff
@@ -138,6 +139,7 @@ refreshSamples = do
 
 -- | Call tidy if available; generate either pretty-printed html or an error message if the html is
 -- invalid.
+-- FIXME: Configurable language instead of fixed `DE`
```

Comment:
Please eliminate this `FIXME` and use `whereToGetTheLangValue` instead of `DE` (which is an implicit `FIXME`).